vcd/config_test.go

```diff
@@ -314,9 +323,155 @@ func TestMain(m *testing.M) {
 		}
 	}
 
+	createSuiteCatalogAndItem(testConfig)
```

Comment:
Add a comment about what we do when we find the item and when we don't.

Reply:
Fixed